charts/lazylibrarian/values.yaml

```diff
@@ -1,153 +1,42 @@
-# Default values for lazylibrarian.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-
-replicaCount: 1
-
-autoscaling:
-  enabled: false
-  minReplicas: 1
-  maxReplicas: 1
+# Default values for LazyLibrarian.
 
 image:
   repository: linuxserver/lazylibrarian
   pullPolicy: IfNotPresent
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: 2551a8bc-ls25
-
-imagePullSecrets: []
-nameOverride: ""
-fullnameOverride: ""
-
-timezone: UTC
-puid: 1001
-pgid: 1001
-
-# Linuxserver.io additional layers.
-# Enables additional features for the image, at the cost of increased size and
-# possible incompatabilities with certain architectures. Disabled by default.
-#
-# To enable, set enabled to true, and follow the guide at: https://github.com/linuxserver/docker-lazylibrarian#application-setup
-# to configure the application.
-dockerMods:
-  # Enable the Calibre Docker Mod to allow Calibredb import
-  calibre:
-    enabled: false
-    image:
-      repository: linuxserver/calibre-web
-      tag: calibre
-  # Enable the FFMpeg Docker Mod. This allows using the audiobook conversion features of LazyLibrarian.
-  ffmpeg:
-    enabled: false
-    image:
-      repository: linuxserver/mods
-      tag: lazylibrarian-ffmpeg
-
-serviceAccount:
-  # Specifies whether a service account should be created
-  create: true
-  # Annotations to add to the service account
-  annotations: {}
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
-  name: ""
-
-podAnnotations: {}
-
-podSecurityContext: {}
-  # fsGroup: 2000
+  tag: version-047f91af
```

Comment:
Is this version the same as `appVersion` in `chart.yaml`? May be better to keep them in sync

Reply:
App version has not changed, latest release is still 1.7.2